crypto/sha/asm/sha512-x86_64.pl

```diff
@@ -2207,9 +2263,14 @@ ()
 
 	jbe	.Loop_avx2
 	lea	(%rsp),$Tbl
+# temporarily use $Tbl as index to $_rsp
+# this avoids the need to save a secondary frame pointer at -8(%rsp)
+.cfi_cfa_expression	$Tbl+`16*$SZ+3*8`,deref,+8
 
 .Ldone_avx2:
 	lea	($Tbl),%rsp
```

Comment:
a branch from the inner loop goes here, $Tbl is the "right" value in both cases.

Reply:
Actually I can remove this instruction if I use the same info as the cfi info with the Tbl.
Guess I will try that.